tests/databricks/test_notebook.py

```diff
@@ -151,7 +151,8 @@ def test_databricks_notebook_operator_without_taskgroup_new_cluster(
             "libraries": [{"nb_index": {"package": "nb_package"}}],
             "timeout_seconds": 0,
             "email_notifications": {},
-        }
+        },
+        version="2.1",
```

Comment:
Can we use a variable instead of hardcoding the api version?

Reply:
yes, just referenced the constant in a newer commit to address this comment.